src/trie.py

```diff
@@ -0,0 +1,93 @@
+"""Module for Trie tree."""
+
```

Comment:
I would not make multi-line comments like this, use the triple """ (that's what they are there for)